src/UI/Component/Item/Factory.php

```diff
@@ -0,0 +1,78 @@
+<?php
+namespace ILIAS\UI\Component\Listing;
+/**
+ * This is how a factory for Items looks like.
+ */
+interface Factory {
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *      The Appointment Item is used to summarize one single
+	 *      appointment in a list.
```

Comment:
Your correct, we should rephrase that to e.g.: "The Appointment Item is used to summarize one single appointment. Depending on the context (e.g. list) the item might look different."